### DISCUSSION TITLE
Okteto Up not able to replace Kubernetes resource (Pod)

### DISCUSSION THREAD

**Discussion by vanja-bisanovic** [okteto/okteto#2258] [Q&A]
Hello,

we are using okteto to allow development on the Kubernetes Cluster that runs locally. 
We first deploy our Microservice and all of its resources using helm upgrade --install ... command. After the service is running,
we call okteto up, expecting that the microservice pod will be replaced by the okteto-manged microservice pod, as it is described in the documentation.
This however is not happening, due to the error "Resource [pod name] already exist".
These are the okteto logs:

```

"'.stignore' exists in folder '/home/vanja/dev/workspaces/ris-administration'" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"Terminal: &{{1280 5 191 35387 0 [3 28 127 21 4 0 1 0 17 19 26 0 18 15 23 22 0 0 0] 0 0}}" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"activating development container retry=false" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"starting shutdown sequence" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"sent cancellation signal" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"stopping forwarders" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"completed shutdown sequence" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"activate failed with: resource administration already exist" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"exit signal received due to error: resource administration already exist" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3

"error deleting deprecated volume: persistentvolumeclaims \"okteto-administration\" not found" action=fc689dfc-1f1d-4942-901b-f7f2a037eea8 version=1.15.3
```

Any ideas would be greatly appreciated.

Best regards, Vanja

**Comment by pchico83** [comment]
@vanja-bisanovic would you mind sharing the content of your okteto manifest? and the output of `kubectl get all`? Looks like there is a conflict creating a resource named `administration`.
